plutus-core/plutus-core.cabal

```diff
@@ -326,6 +299,61 @@ library
         word-array -any,
         cardano-crypto-class -any
 
+-- could split this up if we split up the main library for UPLC/PLC/PIR
```

Comment:
It may be good to split up the main library for UPLC/PLC/PIR......probably should be in a separate PR though.